src/charm.py

```diff
@@ -10,63 +10,90 @@
 import os
 from charms.operator_libs_linux.v2 import snap  # type: ignore
 from ops.model import ActiveStatus, MaintenanceStatus
-from snap_management import SnapSpecError, SnapInstallError, SnapServiceError, install_snap, node_exporter_snap_name, opentelemetry_collector_snap_name
+from snap_management import (
+    SnapSpecError,
+    SnapInstallError,
+    SnapServiceError,
+    install_snap,
+    node_exporter_snap_name,
+    opentelemetry_collector_snap_name,
+)
+from singleton_snap import SingletonSnapManager
 
 # Log messages can be retrieved using juju debug-log
 logger = logging.getLogger(__name__)
-VALID_LOG_LEVELS = ["info", "debug", "warning", "error", "critical"]
+VALID_LOG_LEVELS = ['info', 'debug', 'warning', 'error', 'critical']
```

Comment:
@lucabello mind checking why our `tox -e fmt` is doing this here?

Reply:
Keeping it the way we have it in all charms for now